### DISCUSSION TITLE
Getting Type Error on build in ./node_modules/next-auth/src/core/errors.ts

### DISCUSSION THREAD

**Discussion by vectrobyte** [nextauthjs/next-auth#7386] [Help]
version: 4.22.1

./node_modules/next-auth/src/core/errors.ts:11:11
Type error: No overload matches this call.
  Overload 1 of 2, '(message?: string, options?: ErrorOptions): Error', gave the following error.
    Argument of type 'string | Error' is not assignable to parameter of type 'string'.
      Type 'Error' is not assignable to type 'string'.
  Overload 2 of 2, '(message?: string): Error', gave the following error.
    Argument of type 'string | Error' is not assignable to parameter of type 'string'.
      Type 'Error' is not assignable to type 'string'.

   9 |   constructor(error: Error | string) {
  10 |     // Support passing error or string
> 11 |     super((error as Error)?.message ?? error)
     |           ^
  12 |     this.name = "UnknownError"
  13 |     this.code = (error as any).code
  14 |     if (error instanceof Error) {
error Command failed with exit code 1.

**Comment by jph973** [comment]
Check if you have any imports from "next-auth/src".
I found my WebStorm IDE mistakenly auto-generated an import from "next-auth/src" instead of correctly using "next-auth", resulting in the same error.

**Reply by bsplosion** [reply]
Thanks a ton for posting this - you probably just saved me a couple hours of triage before realizing what was going on.

**Reply by wcisco17** [reply]
Amazing this saved me a ton of time from debugging 👏🏾

**Reply by yoty66** [reply]
You dropped this 👑
